common/production_methods/13_construction.txt

```diff
@@ -14,7 +14,7 @@
 			goods_input_fabric_add = 25
 			goods_input_wood_add = 75
 		}
-
+		#(25x20)+(75x20) = 2000;1/1000
```

Comment:
The numbers, Fallonar! What do they mean!?

Reply:
Sorry, I'll clarify the numbers' XD meaning